etcd/etcd-performance.adoc

```diff
@@ -4,4 +4,31 @@ include::_attributes/common-attributes.adoc[]
 = Performance considerations for etcd
 :context: etcd-performance
 
-// This assembly will contain modules to provide information about etcd performance.
\ No newline at end of file
+toc::[]
+
+To ensure optimal performance and scalability for etcd in {product-title}, you can complete the following practices.
+
+include::modules/etcd-node-scaling.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+* link:https://docs.redhat.com/en/documentation/assisted_installer_for_openshift_container_platform/2024/html/installing_openshift_container_platform_with_the_assisted_installer/expanding-the-cluster#adding-hosts-with-the-api_expanding-the-cluster[Adding hosts with the API]
+* link:https://docs.redhat.com/en/documentation/assisted_installer_for_openshift_container_platform/2023/html/assisted_installer_for_openshift_container_platform/expanding-the-cluster#installing-primary-control-plane-node-healthy-cluster_expanding-the-cluster[Installing a primary control plane node on a healthy cluster]
+* link:https://docs.redhat.com/en/documentation/assisted_installer_for_openshift_container_platform/2024/html/installing_openshift_container_platform_with_the_assisted_installer/expanding-the-cluster#installing-control-plane-node-healthy-cluster_expanding-the-cluster[Expanding the cluster]
+* xref:../backup_and_restore/control_plane_backup_and_restore/disaster_recovery/scenario-2-restoring-cluster-state.adoc#dr-restoring-cluster-state[Restoring to a previous cluster state]
+
+include::modules/move-etcd-different-disk.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+* link:https://docs.openshift.com/container-platform/4.11/architecture/architecture-rhcos.html[Red Hat Enterprise Linux CoreOS (RHCOS)]
```

Comment:
I know it was there in the previous assembly like that, but this link is pointing to an unsupported version of OCP docs (4.11). Can this one get updated to use an xref so that it can always go to the latest version of these docs?